cvsutils/commands/predict_dataset.py

```diff
@@ -48,7 +54,7 @@ def main():
     parser.add_argument('iteration_id', type=uuid.UUID)
     parser.add_argument('input_dataset_filepath', type=pathlib.Path, help="A dataset that contains input images. The labels will be ignored.")
     parser.add_argument('output_directory', type=pathlib.Path)
-    parser.add_argument('--threshold', type=float, default=0.1, help="Probability threshold")
+    parser.add_argument('--threshold', nargs='*', default=[DEFAULT_PROB_THRESHOLD], help="Probability threshold (default=0.1)")
```

Comment:
Using this argument for two purposes is bit confusing. Can we add a new argument, for example --threshold_per_tag TAG_NAME THRESHOLD

Reply:
if we use "action='append'", user can specify --threshold_per_tag multiple times and we can get a list of the given parameters.